SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -220,16 +288,37 @@ Interest InterestFactory::CreatePlayerOwnedActorInterest() const
 
 	AddUserDefinedQueries(LevelConstraints, ClientComponentInterest.Queries);
 
+	const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>();
+	if (SpatialGDKSettings->bEnableNetCullDistanceInterest && SpatialGDKSettings->bEnableNetCullDistanceFrequency)
```

Comment:
Do we need to check both properties? It seems more practical to have the frequency flag override the non-freq one, especially as they both mention net cull distance

Reply:
Yep fair, will update to the later only.